packages/scene-composer/src/components/WebGLCanvasManager.tsx

```diff
@@ -119,10 +119,10 @@ export const WebGLCanvasManager: React.FC = () => {
             <gridHelper
               ref={gridHelperRef}
               args={[
-                1000 /* size */,
-                500 /* grid# */,
-                new THREE.Color('#d5dbdb') /* center line color */,
-                new THREE.Color(hexColorFromDesignToken(awsui.colorTextInputPlaceholder)) /* grid color */,
+                1000 /* size */, 500 /* grid# */,
```

Comment:
Flickering appears to have something to do with modifying the colors. Will need to look into it more so that we can bring back this customization later, but otherwise it should be resolved. 

Totally agree that we should look into not hard-coding the grid size, however, the modification you see here is the linter putting it in a single line.